assets/src/stories-editor/components/higher-order/with-amp-story-settings.js

```diff
@@ -513,7 +513,7 @@ export default createHigherOrderComponent(
 									} }
 								/>
 								<RangeControl
-									label={ __( 'Background Opacity', 'amp' ) }
+									label={ __( 'Opacity', 'amp' ) }
```

Comment:
Are you removing this for translation purposes or generally as well? Wondering if it's clear without it that it's for the background only? I guess it should be.

Reply:
Mainly because it should be clear enough and because we already use `__( 'Opacity', 'amp' )` as a string in the editor. Makes things consistent and results in one string less to translate.